BotwActorTool.GUI/Extensions/ActorInfoExtension.cs

```diff
@@ -38,11 +42,33 @@ public static ObservableCollection<TreeNodeModel> LoadActorInfoNodes(string modR
 
             return new(actorInfo.RootNode.Hash["Actors"].Array.Where(condition).Select(x => {
                 return new TreeNodeModel(x.Hash["name"].String, x.GetActorDescription()) {
-#if !DEBUG // Disable in DEBUG to speed up initialization time, removes 5-10sec
-                    Meta = x.SerializeNode() ?? ""
-#endif
+                    Meta = x
                 };
             }).OrderBy(x => x.Key));
         }
+
+        public static string GetActorDescription(this BymlNode actor)
+        {
+            Dictionary<string, string> schema = new() {
+                { "Name", "WIP" },
+                { "Bfres", "#bfres" },
+                { "Model", "#mainModel" },
+                { "Profile", "#profile" },
+            };
+
+            StringBuilder str = new();
+            foreach ((var name, var value) in schema) {
```

Comment:
The order of the KeyValuePairs in `schema` will probably always be what you expect in this case, but as Dictionary order is not defined, it's better practice to use something else: OrderedDictionary, `(string, string)[]` (array of 2-string tuples), etc.

Since you're never actually using any of the properties of a Dictionary that make it advantageous, I suggest the array of 2-string tuples, but this is called, as I understand it, only once per program run, so dictionary-type or list-type overhead would be a non-issue.